master/internal/user/postgres_users.go

```diff
@@ -131,6 +131,13 @@ func Update(
 				Where("id = ?", updated.ID).Exec(ctx); err != nil {
 				return fmt.Errorf("error setting active status of %q: %s", updated.Username, err)
 			}
+			// Revoke all access tokens of a user when it is deactivated.
+			if !updated.Active {
+				err := revokeUserAccessTokens(ctx, tx, updated.ID)
```

Comment:
what happens if the user being deactivated doesn't have any access tokens?

Reply:
The `Exec` function in Bun will still execute successfully, even if no rows are affected. The absence of matching rows is not considered an error by SQL standards, and Bun will not return an error for this case. Therefore, the function will return `nil`, indicating no errors occurred.

If you want to check whether any rows were affected by the update, you can inspect the result object returned by `Exec`, but IMO, having any log might create a noise in this case. 

```
result, err := tx.NewUpdate().
	Table("user_sessions").
	Set("revoked = ?", true).
	Where("user_id = ?", userID).
	Where("token_type = ?", model.TokenTypeAccessToken).
	Exec(ctx)
if err != nil {
	return err
}

rowsAffected, err := result.RowsAffected()
if err != nil {
	return err
}

if rowsAffected == 0 {
	// No rows were updated, handle accordingly
}
```

Reply:
Perfect! 😄